app/views/transaction_detail/_details.html

```diff
@@ -1,50 +1,58 @@
 <table class="transaction-details">
   <tbody>
+
     <tr>
       <td>Reference number:</td>
       <td id="reference">{{reference}}</td>
     </tr>
+
     {{#permissions.transactions_description_read}}
     <tr>
       <td>Description:</td>
       <td id="description">{{description}}</td>
     </tr>
     {{/permissions.transactions_description_read}}
+
+    <tr>
+      <td>Payment status:</td>
+      <td id="state">{{state_friendly}}{{#state.message}} ({{state.code}} - {{state.message}}){{/state.message}}</td>
+    </tr>
+
     {{#permissions.transactions_amount_read}}
     <tr>
-      <td>Amount:</td>
+      <td>Payment amount:</td>
       <td id="amount">{{amount}}</td>
     </tr>
     {{/permissions.transactions_amount_read}}
+
+    {{#refunded}}
```

Comment:
confirmed with till. We will go with this behaviour for now pending @soniaturcotte getting back